src/lib/style/theme.js

```diff
@@ -82,6 +82,28 @@ export const defaultTheme = {
     border: '#313131',
     info: '#434343',
   },
+  darkRebrand: {
+    statusHealthy: '#0AADA6',
+    statusWarning: '#F8F32B',
+    statusCritical: '#E84855',
+    selectedActive: '#037AFF',
+    highlight: '#1A3C75',
+    border: '#313131',
+    buttonPrimary: '#2F4185',
+    buttonSecondary: '#595A78',
+    buttonDelete: '#3D0808',
+    infoPrimary: '#8E8EAC',
+    infoSecondary: '#333366',
+    backgroundZ1: '#121219',
+    backgroundZ2: '#323245',
+    backgroundZ3: '#232331',
+    backgroundZ4: '#171721',
```

Comment:
I'm not convinced by those labels. IMO we should reflect a bit more their use case in their labels or explicit their hierarchy via a Level label (eg `backgroungLevel1`) .

Reply:
agree, let's take `backgroundLevel1`, etc.